src/components/Counter.test.js

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from "@testing-library/react";
+import Counter from "./Counter";
+import user from "@testing-library/user-event";
+
+describe("Counter", () => {
+  test("renders correctly", () => {
+    render(<Counter />);
+
+    const countElement = screen.getByTestId("counter");
```

Comment:
If it is possible, it is better to avoid targeting elements by testid as is not the most accurate representation of how the end user interacts with a front-end application, nor class names and IDs. For the most part, they interact with the UI elements. Targeting elements by role and more interactive attributes makes for a far better real-world test. It also encourages writing code with good a11y quality.